hopp/simulation/technologies/sites/site_info.py

```diff
@@ -198,13 +198,6 @@ def plot(self,
         axes.set_aspect('equal')
         axes.set(xlim=(min_plot_bound[0], max_plot_bound[0]), ylim=(min_plot_bound[1], max_plot_bound[1]))
         plot_shape(figure, axes, self.polygon, '--', color=border_color, alpha=alpha, linewidth=linewidth / 2)
-        if isinstance(self.polygon, Polygon):
```

Comment:
This is the issue I was describing yesterday, with the `shapely` version clash. We should be able to keep this code, just uninstall `shapely` from conda and make sure it's installed via `requirements.txt`

Reply:
What's happening with this? Are there problems plotting shapely objects?

Reply:
The install instructions for HOPP have this line:

`conda install -c conda-forge shapely==1.7.1 -y`

This version of shapely leads to errors with the current code. We also define `shapely` in `requirements.txt` with a more recent version, but the conda install takes precedent

Reply:
If those errors are fixed, then could this plotting be added back?

Reply:
I will look at adding these lines back in.